backend/db/migrate/20250805144217_add_google_oauth_to_users.rb

```
@@ -0,0 +1,6 @@
+class AddGoogleOauthToUsers < ActiveRecord::Migration[8.0]
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)